strings/word_patterns.py

```diff
@@ -1,5 +1,14 @@
 def get_word_pattern(word: str) -> str:
     """
+    Returns numerical pattern of character appearances in given word
+    >>> get_word_pattern()
+    Traceback (most recent call last):
+    ...
+    TypeError: get_word_pattern() missing 1 required positional argument: 'word'
+    >>> get_word_pattern("")
+    ''
+    >>> get_word_pattern(" ")
+    '0'
```

Comment:
Please put the tests that work correctly first and then afterward the ones that raise exceptions.

Also, please add:
        >>> get_word_pattern(1)
        >>> get_word_pattern(1.1)
        >>> get_word_pattern([])